en/flight_controller/durandal.md

```diff
@@ -0,0 +1,365 @@
+# Holybro Durandal
```

Comment:
This is produced from the various docs they have on site and info from Andries. Should be fairly correct.